docs/source/start-routingspec.rst

```diff
@@ -378,17 +433,17 @@ In the above image there are three parts boxed in red:
 * Sample Message: Box for pasting or loading retrieved messages to test DECODING
 * Decoded Data: Output of DECODING with color syntax
 
-In addition to the parts highligted above, there are three other parts that 
+In addition to the parts highlighted above, there are three other parts that 
 the user may use:
 
-* Sesnor Line Conversions: Table for specifying units of decoded message and any simple linear coefficient conversions
+* Sensor Line Conversions: Table for specifying units of decoded message and any simple linear coefficient conversions
 * Data order:  A drop down menu where Ascending or Descending can be selected (default is undefined)
-* Header Type:  A drop fown menu for selecting a header type such as a medium or source type (default is blank)
+* Header Type:  A drop down menu for selecting a header type such as a medium or source type (default is blank)
 
 In regards to using the sample message browser, note that messages
 can only be retrieved from an lrgs for which a connection has already
 been established.  For USACE users, this means that messages can only
-be retireved while logged onto the server.
+be retrieved while logged onto the server.
```

Comment:
Consider dropping this sentence about USACE users.   It is not clear to me what the 'server' means in this context.

Reply:
I think this will be clear to USACE users - but perhaps this something that could be elaborated on and explained in the Confluence Page - given the relevance to USACE/CWMS?